EPRN.Common.Data/Migrations/20240201160919_Add December Waste.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace EPRN.Waste.API.Migrations
```

Comment:
Just wanted to make sure we are adding this migration in the correct namespace?

Reply:
There are similar previous migrations.